src/store/store.ts

```diff
@@ -579,7 +584,8 @@ export const useStore = defineStore('store', {
         this.getAmalgamatingBusinessesValid &&
         this.isDefineCompanyValid &&
         this.isAddPeopleAndRolesValid &&
-        this.isCreateShareStructureValid
+        this.isCreateShareStructureValid &&
+        this.getFolioNumberValid
```

Comment:
I think this will be false by default, so I think you'll need to make it True if premium account.

Example: https://github.com/bcgov/business-create-ui/blob/0902aa5a7e631ab0abc81567681a77364da6741f/src/store/store.ts#L538

Reply:
Yes

Reply:
Looks good. Resolving.